release/k3s/k3s.go

```diff
@@ -80,6 +80,7 @@ type Release struct {
 	Handler       string `json:"handler"`
 	Email         string `json:"email"`
 	Token         string `json:"token"`
+	SSHKeyPath    string `json:"ssh_key_path"`
```

Comment:
How does this field get populated with the path?

Reply:
The k3s_release utility takes in a config.json file as an argument. It looks like this:
```json
{
"old_k8s_version": "v1.25.4",
"new_k8s_version": "v1.25.5",
"old_k3s_version": "k3s1",
"new_k3s_version": "k3s1",
"old_k8s_client": "0.25.4",
"new_k8s_client": "0.25.5",
"handler": "nikolaishields",
"workspace": "/tmp/dec-release/1.25",
"email": "contact@nikolaishields.com",
"release_branch": "release-1.25",
"token": "xxxx"
}
```

Reply:
Essentially the way this will look after merging this code:
```json
{
"old_k8s_version": "v1.25.4",
"new_k8s_version": "v1.25.5",
"old_k3s_version": "k3s1",
"new_k3s_version": "k3s1",
"old_k8s_client": "0.25.4",
"new_k8s_client": "0.25.5",
"handler": "nikolaishields",
"workspace": "/tmp/dec-release/1.25",
"email": "contact@nikolaishields.com",
"release_branch": "release-1.25",
"token": "xxxx",
"ssh_key_path": "~/.ssh/id_ecdsa" #or whatever the users key is named
}
```